api/src/main/resources/config/application.yaml

```diff
@@ -48,3 +48,9 @@ logging:
     name: ${LOG_FILE_NAME:rosetta-api.log}
     max-size: ${LOG_FILE_MAX_SIZE:10MB}
     max-history: ${LOG_FILE_MAX_HISTORY:10}
+
+management:
+  endpoints:
+    web:
+      exposure:
+        include: "info"
```

Comment:
Will this enable **only** "info" endpoints? What are those? We need to add them to the documentation.

Reply:
Only one info endpoint with information which I posted before

Reply:
So, will we delete it for the production? If so, you could place it into the dev environment profile.